### DISCUSSION TITLE
Is Webpack going anywhere? Turbopack and the future of Webpack

### DISCUSSION THREAD

**Discussion by lannonbr** [webpack/webpack#16418] [Q&A] [answered]
With the announcement of [Turbopack](https://turbo.build/pack) by the team at Vercel, can anyone currently on the Webpack team confirm whether or not Turbopack is a replacement and this is going away or if it is just marketing fluff and Webpack will continue along in development.

**Comment by alexander-akait** [comment, marked as answer]
https://twitter.com/wsokra/status/1585033634427596800?s=46&t=LzBAaPY8H2yoHh_p0cdflA from @sokra (creator of webpack), turbopack is young, need more time to be stable and mature, and many next.js configurations use custom webpack configuration, so webpack will get fixes and improvements

**Reply by andygup** [reply]
@alexander-akait thanks for the reply, is there a link to an updated Webpack Roadmap? The last one I can find was for [2021](https://webpack.js.org/blog/2020-12-08-roadmap-2021/). That would be very helpful for us (and most likely a bunch of others) for planning purposes.

**Reply by alexander-akait** [reply]
Currently we have webpack-6 label, but I think it can be convert them to cards and provide the better roadmap

**Reply by MirKml** [reply]
Thanks Alex (also thanks for your hard work on Webpack ekosystem). From the perspective of releases, webpack had each month one bug fix or minor release. Nothing happens from end of July.
As we saw, Tobias had last months a lot of work on newly announced Rust based Next.js tooling. But it seems that nobody can replace him with his major role in Webpack as maintainer (merging PR, reviews, fix bugs ..) or even in development of new features.
My opinion - I think it's very hard to maintain JS based Webpack and another Rust based Turbo tools with same goals by same person. We will see, but I think Webpack would have lower priority.
Roadmap updates and future plans for Webpack from core team will be very welcome.

**Reply by alexander-akait** [reply]
Yes, in near future we will provide more information and some stale PRs will be merged too with new release
